contracts/ExecutionManager.sol

```diff
@@ -67,42 +86,43 @@ contract ExecutionManager is IExecutionManager, AccessControlExtended {
     /**
      * @notice Function for getting the minimum fees required for executing a cross-chain transaction
      * @dev This function is called at source to calculate the execution cost.
-     * @param msgGasLimit_ Gas limit for the transaction
      * @param siblingChainSlug_ Sibling chain identifier
      * @return Minimum fees required for executing the transaction
      */
     function getMinFees(
-        uint256 msgGasLimit_,
+        uint256,
         uint32 siblingChainSlug_
     ) external view override returns (uint256) {
-        return _getMinExecutionFees(msgGasLimit_, siblingChainSlug_);
+        return executionFees[siblingChainSlug_];
```

Comment:
could be 0 for a Slug that is not configured yet

Reply:
Dont think this affects much.